src/main/java/Duke.java

```diff
@@ -1,10 +1,157 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+    static boolean ifExit = false; //exits program if true
+    static Task[] tasks = new Task[100]; //List of tasks
```

Comment:
just a suggestion: the value 100 can be placed as a named constant so that it's easier for you to change this value (e.g. dh to keep scrolling down the code)